tests/dummy/app/router.js

```diff
@@ -5,13 +5,50 @@ var Router = Ember.Router.extend({
   location: config.locationType
 })
 
+var addRoute = function (routeConfig) {
```

Comment:
Can we put this function into an addon so that we can use it in multiple places?